app/services/malware_scan.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+class MalwareScan
```

Comment:
Should we put this in `app/lib` or namespace under `app/services/azure`?

Reply:
I've moved it to `lib`